content/metadata.yaml

```diff
@@ -549,12 +549,12 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Pusan National University
-# - name: Binglan Li
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - University of Pennsylvania
+- name: Binglan Li
+  orcid: null
```

Comment:
I'm not sure whether `null` is allowed.  We can try merging and see what happens.